driver/Dockerfile.hail

```diff
@@ -18,6 +18,7 @@ RUN apt update && \
     pip3 install \
         analysis-runner \
         bokeh \
+        cloudpathlib \
```

Comment:
AFAIK this won't work properly. It requires the 'extras' to define the cloud environments it will work with. 

https://github.com/populationgenomics/vigilant-winner/blob/main/requirements.txt#L2

Reply:
https://cloudpathlib.drivendata.org/stable/#installation